src/components/List/index.jsx

```diff
@@ -8,8 +8,10 @@ export const Header = ({ children }) => (
   <div className={styles['c-list-header']}>{children}</div>
 )
 
-export const Row = ({ children }) => (
-  <div className={styles['c-list-row']}>{children}</div>
+export const Row = props => (
+  <div ref={props.onRef} className={styles['c-list-row']}>
+    {props.children}
+  </div>
```

Comment:
Nit: it seems it's a good use case for [`React.forwardRef`](https://reactjs.org/docs/forwarding-refs.html)

Reply:
WHAT! Thank you! Didn't know about this one.